.luacheckrc

```diff
@@ -0,0 +1,13 @@
+unused_args = false
+allow_defined_top = true
+
+read_globals = {
+	"DIR_DELIM",
+	"vector", "nodeupdate",
+	"VoxelManip", "VoxelArea",
+	"PseudoRandom", "ItemStack",
+}
+globals = { "minetest", "core" }
```

Comment:
It's not supposed to be used outside of builtin. Doesn't matter that much though